package.json

```
@@ -46,6 +46,6 @@
   "dependencies": {
     "jose": "4.9.2",
     "node-rsa": "1.1.1",
-    "openid-client": "5.1.5"
+    "openid-client": "^5.4.0"
```

we currently pin all our dependency versions to avoid bugs and breaking changes when upstream packages don't respect semver

```suggestion
    "openid-client": "5.4.0"
```

---

@mantariksh Fixed in latest commit, thank you! 😸